libs/net/dal/Configuration/TagConfiguration.cs

```diff
@@ -3,11 +3,11 @@ namespace TNO.DAL.Configuration;
 using Microsoft.EntityFrameworkCore.Metadata.Builders;
 using TNO.Entities;
 
-public class TagConfiguration : BaseTypeConfiguration<Tag, string>
+public class TagConfiguration : BaseTypeConfiguration<Tag, int>
 {
     public override void Configure(EntityTypeBuilder<Tag> builder)
     {
-        builder.Property(m => m.Id).IsRequired().HasMaxLength(6).ValueGeneratedNever();
+        builder.Property(m => m.Code).IsRequired().HasMaxLength(15).ValueGeneratedNever();
```

Comment:
Add unique index for tag.  I don't think they should be able to create duplicates,

`builder.HasIndex(m => m.Tag, "IX_tag").IsUnique();`

Reply:
@jdtoombs I don't see the unique indexing being added here?